src/settings.plugin.js

```diff
@@ -173,24 +172,26 @@ wikiaJWPlayerSettingsPlugin.prototype.createAutoplayToggle = function () {
 
 	return autoplayToggle;
 };
+// end autoplay
 
-wikiaJWPlayerSettingsPlugin.prototype.createQualityLevelsList = function () {
-	var playerInstance = this.player,
-		backIcon = this.createSVG(wikiaJWPlayerIcons.back);
+// quality
```

Comment:
This comment is either not needed or it should be extended with some useful information :)